### PR TITLE
Fix #3557 Merge profile chooser fragment into 2 fargments

### LINKED ISSUE #3557: Merge profile_chooser_fragment.xml into single xml file
Currently there are 4 versions of `profile_chooser_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

It can happen that it becomes really hard to merge all 4 files into single file, in such cases prefer code-readability over single file. Therefore it can be fine if in the end the code still contains 2 or 3 files.

**Note**: **In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.**

### PR DESCRIPTION
**Pull request by priyatanu**
## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [ ] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by priyatanu** [comment]
Hi kindly review @ARJUPTA @rt4914 ,

**Comment by rt4914** [comment]
@priyatanu Please see the above check list and fill the title, description accordingly. 
1. Your title should contain issue number in the format mentioned in checklist
2. Your description should contain issue number in the format mentioned in checklist
3. As per the details in the issue that you are working on you should add screenshots of the before/after screen for the files that you have worked on.

File: app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
           android:id="@+id/profile_select_text"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginStart="64dp"
+          android:layout_marginStart="@dimen/profile_chooser_fragment_layout_marginStart"
```
Comment (rishidyno) [inline app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml:60]: @priyatanu dimension name should be `profile_chooser_fragment_layout_margin_start` instead.
And same goes for others.

**Review by rishidyno — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml (hunk 1 above)
Comment (priyatanu) [reply app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml:60]: thank you for the review. I will update the files

**Comment by priyatanu** [comment]
> 1. our title should contain issue number in the format mentioned in checklist

Thank you i will add the changes

**Comment by priyatanu** [comment]
Reopen the PR

**Pull request closed without merging**